string node_name
bool activate
